src/index.js

```diff
@@ -10,11 +10,16 @@ export default function (contents) {
     external: RegExp().test.bind(/^(.(?!\.(js|es)$))*$/)
   }
 
-  const plugins = this.options.rollup || loaderUtils.parseQuery(this.query)
```

Comment:
looks like we can simply replace this line with `loaderUtils.getLoaderConfig(this, 'rollup')`